tensorrt_llm/tokenizer/tokenizer.py

```
@@ -449,4 +450,20 @@ def load_hf_tokenizer(model_dir: str,
+    # The model may be gated and the files are already cached locally but no
+    # token is available in the current environment. Retry with local cache.
+    try:
+        tokenizer = TransformersTokenizer.from_pretrained(
+            model_dir, local_files_only=True, **load_kwargs)
+
+        if trust_remote_code:
+            maybe_register_transformers_modules_by_value()
+
+        return tokenizer
+
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)